unreleased.rst

```diff
@@ -9,6 +9,12 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
++ [JSON API - Experimental] Flattening the output of the ``/contracts/search`` endpoint.
+  The endpoint returns ``ActiveContract`` objects without ``GetActiveContractsResponse`` wrappers.
+  See `issue #2987 <https://github.com/digital-asset/daml/pull/2987>`_.
+- [DAML Assistant] ``daml start`` now supports ``--sandbox-option=opt``, ``--navigator-option=opt``
```

Comment:
Is it documented how to use this anywhere? Either the in the CLI itself or docs? If not, please add.

Reply:
yes, it is.